packages/server/graphql/mutations/addReactjiToReactable.ts

```diff
@@ -164,13 +165,16 @@ const addReactjiToReactable = {
 
     const data = {reactableId, reactableType}
     const {meetingType} = await dataLoader.get('newMeetings').load(meetingId)
+    const isAIComment = reactable instanceof Comment && reactable.createdBy === PARABOL_AI_USER_ID
     analytics.reactjiInteracted(
       viewerId,
       meetingId,
       meetingType,
       reactableId,
       reactableType,
-      !!isRemove
+      reactji,
+      !!isRemove,
+      isAIComment
```

Comment:
I suggest that we record which `reactji` is being interacted with, and I've added a boolean `isAIComment` field. 

Emojis isn't a perfect way to measure whether users like the summaries, but the types of emojis they use should give us some indication of whether they like it or not.

Reply:
I like this a lot!